installer/database/migrations/migrate_0.5.9.sql

```
@@ -200,6 +200,23 @@ SET @c := (SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA=DA
 SET @s := IF(@c=0, "ALTER TABLE archival_units ADD COLUMN document_filename VARCHAR(255) NULL", 'SELECT 1');
 PREPARE st FROM @s; EXECUTE st; DEALLOCATE PREPARE st;
 
+-- archival_units — phase 7: interoperability standards (IIIF, ARK, RightsStatements)
+SET @c := (SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA=DATABASE() AND TABLE_NAME='archival_units' AND COLUMN_NAME='iiif_manifest_url');
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)